nixos/modules/services/hardware/hddfancontrol.nix

```diff
@@ -0,0 +1,67 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.hddfancontrol;
+  types = lib.types;
+in
+
+{
+  options = {
+
+    services.hddfancontrol.enable = lib.mkEnableOption "hddfancontrol daemon";
```

Comment:
```suggestion
    services.hddfancontrol.enable = lib.mkEnableOption (lib.mdDoc "hddfancontrol daemon");
```

Reply:
ooh missed this - and I'm sorry I did not see your comments before I clicked merge.  I'll open another PR with these tweaks.

Reply:
It seems I comment and you merge it at the same time :)

Reply:
https://github.com/NixOS/nixpkgs/pull/251184